### PR TITLE
Revert workflow disable commit

### PR DESCRIPTION
**Pull request by wheelsandcogs**

### CODE REVIEW

**Comment by lennym** [comment]
You'll likely want this first: https://github.com/UKHomeOffice/taskflow/pull/8

File: lib/hooks/resolve/index.js [hunk 1]
```diff
@@ -31,12 +32,11 @@ module.exports = settings => {
...
-    //   // insta-resolve
-    //   if (model.status === 'new') {
-    //     return pollChangelog(response.MessageId);
-    //   }
-    // });
+      .then(response => {
+        // insta-resolve
+        if (model.status === 'new') {
+          return pollChangelog(response.MessageId);
+        }
```
Comment (lennym) [inline lib/hooks/resolve/index.js]: I don't think this conditional should be there. Shouldn't we always wait for the message to resolve?

Comment (joefitter) [reply lib/hooks/resolve/index.js]: Yeah you're probably right. I was only doing it for things that went straight from `new` to `done`, but thinking about it we probably also want to wait for the reply if a case is resolved by some other means

Comment (lennym) [reply lib/hooks/resolve/index.js]: _nods_

File: lib/hooks/resolve/index.js [hunk 2]
```diff
@@ -32,11 +33,5 @@ module.exports = settings => {
   return model => {
     return queue(model.data)
       .then(response => pollChangelog(response.MessageId));
```
Comment (wheelsandcogs) [inline lib/hooks/resolve/index.js]: @lennym @joefitter so leave it like this?

Comment (joefitter) [reply lib/hooks/resolve/index.js]: this comment seems like it's in response to something, although I don't know what that is :)

Comment (wheelsandcogs) [reply lib/hooks/resolve/index.js]: Click the "show outdated" link above to reopen the previous convo.

Comment (joefitter) [reply lib/hooks/resolve/index.js]: ah yeah, leave it without the `if`

Comment (joefitter) [reply lib/hooks/resolve/index.js]: soz I didn't know this comment was in reference to removing that

Comment (wheelsandcogs) [reply lib/hooks/resolve/index.js]: no worries :D 

Am I good to merge then?

**Pull request merged**